### PR TITLE
Spelling Fixes

### PR DESCRIPTION
**Pull request by klorpa**
## About The Pull Request
Fixes errors in spelling, grammar, and punctuation.
## Why It's Good For The Game
Improves user experience.
## Changelog
:cl:
spellcheck: fixed a few typos
/:cl:

### CODE REVIEW

File: code/modules/unit_tests/preference_species.dm [hunk 1]
```diff
@@ -14,16 +14,16 @@
 		var/species_type = GLOB.species_list[species_id]
 		var/datum/species/species = new species_type()
 
-		// Check the species decription.
+		// Check the species decryption.
```
Comment (BlueMemesauce) [inline code/modules/unit_tests/preference_species.dm:17]: I think this is meant to be description?

Comment (BlueMemesauce) [inline code/modules/unit_tests/preference_species.dm:17]:
```suggestion
		// Check the species description.
```

**Review by E231kei500bandai — APPROVED** [review]

**Comment by mc-oofert** [comment]
> improves user experience

touches mostly code comments

File: code/datums/components/crafting/ranged_weapon.dm [hunk 2]
```diff
@@ -20,7 +20,7 @@
 	category = CAT_WEAPON_RANGED
 
 /datum/crafting_recipe/reciever
-	name = "Modular Rifle Reciever"
+	name = "Modular Rifle Receiver"
```
Comment (jlsnow301) [inline code/datums/components/crafting/ranged_weapon.dm:23]: please get the rest of the datum

File: code/datums/saymode.dm [hunk 3]
```diff
@@ -30,15 +30,15 @@
 	user.log_talk(message, LOG_SAY, tag="changeling [ling_sender.changelingID]")
 	var/msg = span_changeling("<b>[ling_sender.changelingID]:</b> [message]")
 
-	//the recipients can recieve the message
+	//the recipients can receive the message
 	for(var/datum/antagonist/changeling/ling_reciever in GLOB.antagonists)
```
Comment (jlsnow301) [inline code/datums/saymode.dm:34]: receiver here, too

File: code/game/objects/structures/deployable_turret.dm [hunk 4]
```diff
@@ -222,7 +222,7 @@
 
 /obj/machinery/deployable_turret/hmg
 	name = "heavy machine gun turret"
-	desc = "A heavy calibre machine gun commonly used by Nanotrasen forces, famed for it's ability to give people on the recieving end more holes than normal."
```
Comment (jlsnow301) [inline code/game/objects/structures/deployable_turret.dm:225]: calibre is ok here, recieving is not

File: code/modules/client/client_colour.dm [hunk 5]
```diff
@@ -9,9 +9,9 @@
  * so that the "highest priority" one is always displayed as opposed to the default of
  * "whichever was set last is displayed".
  *
- * Refactored to allow multiple overlapping client colours
```
Comment (jlsnow301) [inline code/modules/client/client_colour.dm:12]: i'd prefer these left in, changing UK to US spelling is a can of worms. same goes for behaviour, I hate it but US/UK is too fine a hair to split

File: code/__HELPERS/atmospherics.dm [hunk 6]
```diff
@@ -105,13 +105,13 @@ GLOBAL_LIST_EMPTY(gas_handbook)
...
 				if(factor == "Temperature" || factor == "Pressure")
-					factor_info["tooltip"] = "Reaction is influenced by the [lowertext(factor)] of the place where the reaction is occuring."
+					factor_info["tooltip"] = "Reaction is influenced by the [lowertext(factor)] of the place where the reaction is occurring."
 				else if(factor == "Energy")
 					factor_info["tooltip"] = "Energy released by the reaction, may or may not result in linear temperature change depending on a slew of other factors."
 				else if(factor == "Radiation")
 					factor_info["tooltip"] = "This reaction emits dangerous radiation! Take precautions."
 				else if (factor == "Location")
-					factor_info["tooltip"] = "This reaction has special behaviour when occuring in specific locations."
+					factor_info["tooltip"] = "This reaction has special behavior when occurring in specific locations."
```
Comment (dragomagol) [inline code/__HELPERS/atmospherics.dm:114]:
```suggestion
					factor_info["tooltip"] = "This reaction has special behaviour when occurring in specific locations."
```

Comment (dragomagol) [reply code/__HELPERS/atmospherics.dm:114]: Not a typo

File: code/__HELPERS/atmospherics.dm [hunk 7]
```diff
@@ -138,13 +138,13 @@ GLOBAL_LIST_EMPTY(gas_handbook)
...
```
Comment (dragomagol) [inline code/__HELPERS/atmospherics.dm:147]:
```suggestion
					factor_info["tooltip"] = "This reaction has special behaviour when occurring in specific locations."
```
Same as above

File: code/game/atoms.dm [hunk 8]
```diff
@@ -558,7 +558,7 @@
 /**
  * React to an EMP of the given severity
  *
- * Default behaviour is to send the [COMSIG_ATOM_PRE_EMP_ACT] and [COMSIG_ATOM_EMP_ACT] signal
+ * Default behavior is to send the [COMSIG_ATOM_PRE_EMP_ACT] and [COMSIG_ATOM_EMP_ACT] signal
```
Comment (dragomagol) [inline code/game/atoms.dm:561]:
```suggestion
 * Default behaviour is to send the [COMSIG_ATOM_PRE_EMP_ACT] and [COMSIG_ATOM_EMP_ACT] signal
```

File: code/game/atoms.dm [hunk 9]
```diff
@@ -576,7 +576,7 @@
 /**
  * React to a hit by a projectile object
  *
- * Default behaviour is to send the [COMSIG_ATOM_BULLET_ACT] and then call [on_hit][/obj/projectile/proc/on_hit] on the projectile.
+ * Default behavior is to send the [COMSIG_ATOM_BULLET_ACT] and then call [on_hit][/obj/projectile/proc/on_hit] on the projectile.
```
Comment (dragomagol) [inline code/game/atoms.dm:579]:
```suggestion
 * Default behaviour is to send the [COMSIG_ATOM_BULLET_ACT] and then call [on_hit][/obj/projectile/proc/on_hit] on the projectile.
```

Comment (dragomagol) [reply code/game/atoms.dm:579]: You get the idea

File: code/game/atoms.dm [hunk 10]
```diff
@@ -1080,16 +1080,16 @@
...
  */
 /atom/proc/on_log(login)
 	if(loc)
 		loc.on_log(login)
 
 
 /*
-	Atom Colour Priority System
-	A System that gives finer control over which atom colour to colour the atom with.
+	Atom Color Priority System
```
Comment (dragomagol) [inline code/game/atoms.dm:1091]: Ditto here

**Review by dragomagol — CHANGES_REQUESTED** [review]
Please don't change UK spelling to American

**Comment by optimumtact** [comment]
english vs american spellings are not spelling mistakes and it's frowned upon to fix them in bulk.

You can only change htem when you're modifying code and need to modify the grammar.

This PR is also far too big to reliably review to see how many similar issues you've made.

On top of that, i'm not really super fussed about spelling in comments in the code.

please break this into smaller more reviewable chunks (20-30 sets of changes at a time).

or focus on some other more productive use of everyone's time.

**Pull request closed without merging**